### ISSUE TITLE
test results

### ISSUE THREAD

**Issue by LiangXiaoguo** [songguocode/TADAM#9]
Thank you very much for your excellent work! I tried to add "-- which_set: test" to the ". yaml "file to get the test results, but failed

**Comment by songguocode** [comment]
Tensorboard is not supported in the released version. Please add it to the training code according to your need.

**Comment by LiangXiaoguo** [comment]
I am very sorry for not replying to your message in time, I have successfully solved the problem of the test set. Thank you very much for your excellent work.

**Comment by csgeqian** [comment]
Hello, may I have a look at your tensorboard code? Thank you very much

**Comment by csgeqian** [comment]
> 非常抱歉没有及时回复您的留言，我已经成功解决了测试集的问题。非常感谢您的出色工作。

Hello, may I have a look at your tensorboard code? Thank you very much

**Comment by csgeqian** [comment]
> I am very sorry for not replying to your message in time, I have successfully solved the problem of the test set. Thank you very much for your excellent work.

Hello, may I have a look at your tensorboard code? Thank you very much
